pkg/framework/version_bundle.go

```diff
@@ -6,6 +6,7 @@ import (
 	"log"
 	"sort"
 
+	"github.com/blang/semver"
```

Comment:
I like the use of the semver library here.

Reply:
I'm kind of late here, but is there something specific in that library compared to `github.com/coreos/go-semver/semver` used in `versionbundle`? FYI: There's also same implementation in `versionbundle` library: https://github.com/giantswarm/versionbundle/blob/master/releases_sort.go#L9 :)

Reply:
Absolutely, I can take care of that, already put the changes here to take advantage of that.